packages/shared/src/modules/users/libs/validation-schemas/user-reset-password.validation-schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+
+import { UserValidationMessage, UserValidationRule } from "../enums/enums.js";
+
+type UserSignInRequestValidationDto = {
+	newPassword: z.ZodString;
+};
+
+const userResetPassword = z
+	.object<UserSignInRequestValidationDto>({
+		newPassword: z
+			.string()
+			.trim()
+			.min(UserValidationRule.NON_EMPTY_STRING_MIN_LENGTH, {
+				message: UserValidationMessage.PASSWORD_REQUIRE,
+			})
+			.max(UserValidationRule.PASSWORD_MAX_LENGTH, {
+				message: UserValidationMessage.PASSWORD_MAX_LENGTH,
+			})
+			.min(UserValidationRule.PASSWORD_MIN_LENGTH, {
+				message: UserValidationMessage.PASSWORD_MIN_LENGTH,
+			}),
```

Comment:
There are 2 min values, should be only 1

Reply:
Not fixed

Reply:
I thought that this comment was about userValidationMessage enum having two distinct messages for minLength(which was the case by the way). Will fix this now.